docs/getting-started/explore-intro.asciidoc

```diff
@@ -2,7 +2,7 @@
 
 = Explore
 
-The following section includes an overview of the *Hosts*, *Network*, and *Users* pages.
+The following section includes an overview of the *Hosts*, *Network*, *Users*, and *Threat Intelligence* pages.
```

Comment:
```suggestion
The following section includes an overview of the *Hosts*, *Network*, and *Users* pages.
```

Reply:
AFAIK, the TI page isn't be released for 8.4. I noticed it was visible in previous BCs of 8.4 and ESTEC builds but I'm not seeing it in the latest cloud deployment Joe created (which is running BC4 I believe) or my own ESTEC instance (which is running the latest 8.4 snapshot).

Reply:
Ah, good to know. Thanks!